src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -187,7 +187,9 @@ def type_value(device)
 
       def filesystem_label_value(device)
         fs = filesystem(device)
-        # fs may be nil or a file system not supporting labels, like NFS
+        return "" if fs.nil?
+        return "" if fs.respond_to?(:multidevice?) && fs.multidevice?
```

Comment:
As commented in IRC, we have to show the label when the device is a multidevice filesystem.